Cairo Desktop/CairoDesktop.Common/Localization/Language.cs

```diff
@@ -1,4094 +1,4097 @@
-            { "sSettings_Desktop_BackgroundType_cairoVideoWallpaper", "Wideo" },
-            { "sSettings_Desktop_BackgroundType_bingWallpaper", "Zdjęcie Dnia Bing" },
-            { "sSettings_Desktop_Background_Path", "Ścieżka:" },
-            { "sSettings_Desktop_Background_Style", "Styl:" },
-            { "sSettings_Desktop_BackgroundStyle_Tile", "Kafelki" },
-            { "sSettings_Desktop_BackgroundStyle_Center", "Wyśrodkuj" },
-            { "sSettings_Desktop_BackgroundStyle_Fit", "Dopasuj" },
-            { "sSettings_Desktop_BackgroundStyle_Fill", "Wypełnij" },
-            { "sSettings_Desktop_BackgroundStyle_Stretch", "Rozciągnij" },
-            { "sSettings_Desktop_BackgroundStyle_Span", "Rozepnij" },
-            { "sSettings_IconSize", "Rozmiar ikon:" },
-            { "sSettings_IconSizeLarge", "Duże" },
-            { "sSettings_IconSizeMedium", "Średnie" },
-            { "sSettings_IconSizeSmall", "Małe" },
-            { "sSettings_Taskbar_EnableTaskbar", "Włącz pasek zadań" },
-            { "sSettings_Taskbar_TaskbarEdge", "Pozycja paska zadań:" },
-            { "sSettings_PositionBottom", "Na dole" },
-            { "sSettings_PositionTop", "Na górze" },
-            { "sSettings_Taskbar_DisplayMode", "Tryb wyświetlania:" },
-            { "sSettings_Taskbar_DisplayModeAppBar", "Pasek zadań rezerwuje przestrzeń" },
-            { "sSettings_Taskbar_DisplayModeOverlap", "Pasek zadań nachodzi na okna" },
-            { "sSettings_Taskbar_DisplayModeAutoHide", "Pasek zadań chowa się automatycznie" },
-            { "sSettings_Taskbar_EnableTaskbarMultiMon", "Pokaż pasek zadań na wielu monitorach" },
-            { "sSettings_Taskbar_FullWidthTaskBar", "Użyj całej szerokości monitora dla paska zadań" },
-            { "sSettings_Taskbar_MiddleClick", "Akcja środkowego przycisku myszy:" },
-            { "sSettings_Taskbar_MiddleClickNewWindow", "Otwórz nowe okno" },
-            { "sSettings_Taskbar_MiddleClickCloseWindow", "Zamknij okno" },
-            { "sSettings_Taskbar_EnableThumbnails", "Pokaż miniatury okna" },
-            { "sSettings_Advanced_LoggingLevel", "Poziom logów:" },
-            { "sSettings_Advanced_Shell", "Powłoka" },
-            { "sSettings_Advanced_CairoIsShell", "Cairo jest aktualnie ustawiony jako powłoka systemu." },
-            { "sSettings_Advanced_ExplorerIsShell", "Eksplorator Windows jest aktualnie ustawiony jako powłoka systemu." },
-            { "sSettings_Advanced_SetCairoAsShell", "Ustaw Cairo jako powłokę systemu" },
-            { "sSettings_Advanced_SetExplorerAsShell", "Ustaw Eksplorator Windows jako powłokę systemu" },
-            { "sSettings_Advanced_ShellChanged", "Zmieniono ustawienia powłoki systemu" },
-            { "sSettings_Advanced_ShellChangedText", "Musisz się wylogować, aby zastosować zmiany." },
-            { "sSettings_Advanced_LogOffNow", "Wyloguj się teraz" },
-            { "sSettings_Advanced_LogOffLater", "W
```

Comment:
Seems weird keeping this in a separate class if it is just one language, maybe it makes more sense to add this to `DisplayString` as `DefaultLanguage`.